packages/core-data/src/selectors.ts

```diff
@@ -895,13 +1038,14 @@ export const hasFetchedAutosaves = createRegistrySelector(
  * );
  * ```
  *
- * @param {Object} state Editor state.
+ * @param  state Editor state.
  *
- * @return {*} A value whose reference will change only when an edit occurs.
+ * @return A value whose reference will change only when an edit occurs.
  */
 export const getReferenceByDistinctEdits = createSelector(
-	() => [],
-	( state ) => [
+	// This unused state argument is listed here for the documentation generating tool (docgen).
```

Comment:
is this something we need to update in the docgen tool? is this necessary after the updates in #40236

Reply:
Potentially.

Our tooling sees `@param  state Editor state.` in jsdoc but don't see one in the first argument to createSelector because we only use it in the second argument. It would be nice to handle, but that's also kind of a corner case.

Reply:
Ah okay, yeah I see where there's an incompleteness with the `createSelector` exemption. I think we can resolve this and I think the approach in #40865 is reasonable enough.